Widescreen patch
Twin Castle, Main Hall (Daytime)
Left balcony knight
Retail: Hides knight based on camera offset.
Fix: Always show, never hide.
--- original
+++ modified
@@ -426,7 +426,6 @@
 jmp_cmp >, 0xffffffc4, stor[25], inl[:LABEL_70]
 jmp inl[:LABEL_69]
 LABEL_70:
-call 106, 0x1
 yield
 call 257, stor[24], stor[25]
 jmp_cmp >, 0xfffffff3, stor[24], inl[:LABEL_70]
